main.py

```diff
@@ -211,6 +227,25 @@ def run_inventory_loop(config, connection):
 
     call_args = get_args()
 
+    NETBOX_ENVIRONMENT = call_args.netbox
+    url_netbox_device_q = f"https://netbox.{NETBOX_ENVIRONMENT}.cloud.sap/api/dcim/devices/?q="
+    url_netbox_ip_device = f"https://netbox.{NETBOX_ENVIRONMENT}.cloud.sap/api/ipam/ip-addresses/?device="
+    url_netbox_device = f"https://netbox.{NETBOX_ENVIRONMENT}.cloud.sap/api/dcim/devices/"
+    url_netbox_device_interface = f"https://netbox.{NETBOX_ENVIRONMENT}.cloud.sap/api/dcim/interfaces/"
+
+    args_user_name = call_args.username
+    args_password = call_args.password
+    args_write_flag = call_args.write
+    args_force_flag = call_args.force
+    args_iponly_flag = call_args.iponly
+    args_mtu = call_args.mtu
+
+    if args_force_flag:
+        args_write_flag = True
+    is_apod = True if "ap" in str.lower(call_args.query) else False
+        
+    inventory_obj = InventoryContext(url_netbox_device_q, url_netbox_ip_device, url_netbox_device, url_netbox_device_interface, args_user_name, args_password, args_write_flag, args_force_flag, args_iponly_flag, args_mtu, is_apod)    
```

Comment:
linters don't like those long lines. The max is usually 120 characters.

Use this format instead

```
inventory_obj = InventoryContext(
    url_netbox_device_q,
    url_netbox_ip_device,
    url_netbox_device,
    url_netbox_device_interface,
    args_user_name,
    args_password,
    args_write_flag,
    args_force_flag,
    args_iponly_flag,
    args_mtu,
    is_apod
)
```